presto-server-rpm/pom.xml

```diff
@@ -33,143 +35,159 @@
+                                    <destination>/etc/init.d</destination>
+                                    <base>dist/etc/init.d</base>
+                                    <!-- make sure init.d scripts are executable -->
+                                    <fileMode>0755</fileMode>
+                                    <includes>
+                                        <include>*</include>
+                                    </includes>
+                                </rule>
+
+                                <rule>
+                                    <!-- This should go to just /usr/lib/presto eventually. But that needs modifying
+                                    launcher.py in airlift, to have a configurable option for install_path -->
+                                    <destination>/usr/lib/presto/lib</destination>
+                                    <base>${server.tar.package}/lib</base>
+                                    <includes>
+                                        <include>*</include>
+                                    </includes>
+                                </rule>
+
+                                <rule>
+                                    <destination>/usr/lib/presto/lib/plugin</destination>
+                                    <base>${server.tar.package}/plugin</base>
+                                    <includes>
+                                        <include>*/*</include>
+                                    </includes>
+                                </rule>
+
+                                <rule>
+                                    <destination>/etc/presto</destination>
+                                    <base>dist/config</base>
+                                    <includes>
+                                        <include>*</include>
+                                    </includes>
+                                </rule>
+
+                                <rule>
+                                    <destination>/usr/shared/doc/presto</destination>
+                                    <base>${server.tar.package}</base>
+                                    <includes>
+                                        <include>README.txt</include>
+                                    </includes>
+                                </rule>
+
+                                <!-- Add these rules so that .spec knows these dirs are to be removed too on rpm -e -->
+                                <rule>
+                                    <destination>/usr/lib/presto</destination>
+                                </rule>
+                                <rule>
+                                    <destination>/usr/lib/presto/lib</destination>
+                                </rule>
+
+                            </rules>
+                        </package>
+                    </packages>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     <profiles>
```

Comment:
This is the other bit that is changed.  It's the profile added for old versions of rpm